github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRPullRequest.java

```diff
@@ -60,7 +62,7 @@ public GitHubPRPullRequest(GHPullRequest pr) throws IOException {
         userLogin = pr.getUser().getLogin();
         number = pr.getNumber();
         prUpdatedAt = pr.getUpdatedAt();
-        issueUpdatedAt = pr.getIssueUpdatedAt();
+        issueUpdatedAt = pr.getUpdatedAt();
```

Comment:
https://github.com/kohsuke/github-api/issues/178
Don't change because of wrong deprecation.
github-api is very bad maintained, kohsuke randomly do changes.